src/quo2/components/list_items/preview_list/view.cljs

```diff
@@ -0,0 +1,181 @@
+(ns quo2.components.list-items.preview-list.view
+  (:require [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.markdown.text :as quo2.text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]
+            [react-native.hole-view :as hole-view]))
+
+(def params
+  {:size/s-32 {:size          32
+               :border-radius {:rounded 16 :squared 10}
```

Comment:
should we put these in their own file? some properties or styles file? 🤔

Reply:
I left these `params` as it is because it's not the actual style.

But, I can move it to style.cljs as it is a number that denotes the size 👍

Reply:
Yep, in button I used a properties file. I think @mohsen-ghafouri did something similar in slide button - maybe we can have another common file for pieces like this which are not quite styles and cleaner out of the view